package.json

```diff
@@ -35,6 +35,7 @@
     "build:test:segments": "node -e \"var b=require('./scripts/segments-data.js'); b.build();\"",
     "watch": "run-p watch:js watch:test:manifest watch:test:segments watch:test:js watch:js:switcher",
     "watch:js": "rollup -c scripts/rollup.config.js -w",
+    "watch:webworker": "rollup -c scripts/webworker.rollup.config.js -w",
```

Comment:
webworkers were not being watched for rebuild, which made mux.js dev work annoying, this makes it so that we watch and rebuild them.

Reply:
nice!